src/Header/Age.php

```diff
@@ -73,7 +73,7 @@ public function getFieldValue()
      * Set number of seconds
      *
      * @param int $delta
-     * @return RetryAfter
+     * @return Age
```

Comment:
`self` is generally better, as it survives renames. Not really important though.